### ISSUE TITLE
Cannot add slides to an empty slider

### ISSUE THREAD

**Issue by mathbr** [iosscripts/iosslider#201]
The way the API method `addSlide` currently works, it is impossible to add slides to an empty slider:

``` javascript
if(!data.settings.infiniteSlider) {

  if(slidePosition <= data.numberOfSlides) {
    $(data.scrollerNode).children(':eq(' + (slidePosition - 1) + ')').before(slideNode);
  } else {
    $(data.scrollerNode).children(':eq(' + (slidePosition - 2) + ')').after(slideNode);
  }

  if($this.data('args').currentSlideNumber >= slidePosition) {
    $this.data('args').currentSlideNumber++;
  }

} else { // ...
```

If `slidePosition` is 1 while there are currently no slides, the else branch is taken which results in `$(data.scrollerNode).children(':eq(-1)').after(slideNode)`. However the same issue applies to the if branch. There are no children, thus the slide node can never be added.

A possible fix is to simply `append()` the slide node to the scroller node if `data.numberOfSlides` is 0.

**Comment by marcwhitbread** [comment]
If you have no slides in the slider you cant expect iosSlider to initialize properly. This is a corner case but I'll see if I can get it working.

Just as you mentioned, the workaround would be using append() then calling $('.iosSlider).iosSlider('update'); to catch the change.

**Comment by marcwhitbread** [comment]
Fixed issue in 1.1.61. Let me know if that fixes your issue.

**Comment by mathbr** [comment]
Yes, the issue is fixed, thanks.
